controls/SV-257842.rb

```diff
@@ -17,14 +17,22 @@
 $ sudo dnf install s-nail'
   impact 0.5
   ref 'DPMS Target Red Hat Enterprise Linux 9'
-  tag check_id: 'C-61583r925511_chk'
+  tag check_id: 'C-61583r942958_chk'
   tag severity: 'medium'
   tag gid: 'V-257842'
-  tag rid: 'SV-257842r925513_rule'
+  tag rid: 'SV-257842r942959_rule'
   tag stig_id: 'RHEL-09-215095'
   tag gtitle: 'SRG-OS-000363-GPOS-00150'
   tag fix_id: 'F-61507r925512_fix'
   tag 'documentable'
   tag cci: ['CCI-001744']
   tag nist: ['CM-3 (5)']
+  tag 'host'
+  tag 'container'
+
+  mail_package = input('mail_package')
```

Comment:
the value for this input in the inspec.yml is 'mailx' but the control says 's-nail'.  we should either better document the reason for our override in the default values (and probably have 's-nail' there but commented out) or correct the default value to be 's-nail'.